src/test/java/org/broadinstitute/hellbender/tools/exome/ACNVCopyRatioModellerUnitTest.java

```diff
@@ -101,13 +105,13 @@ private static double relativeError(final double x, final double xTrue) {
      * </p>
      */
     @Test
-    public void testRunMCMCOnCopyRatioSegmentedModel() {
+    public void testRunMCMCOnCopyRatioSegmentedModel() throws IOException {
```

Comment:
If the test doesn't `throw` you have to wrap `ReadCountCollectionUtils.parse` in a `try. . . catch`, which strikes me as an annoying thing to do in testing code.  If you have a different opinion on the lesser of these two evils I will adopt it.